Gemfile

```diff
@@ -3,10 +3,12 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.0.0'
 
+gem 'pg'
+
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 6.1.1'
 # Use sqlite3 as the database for Active Record
-gem 'sqlite3', '~> 1.4'
+# gem 'sqlite3', '~> 1.4'
```

Comment:
Let's just remove this gem since we won't be using it anymore. Also, make sure you perform a `bundle install` after removing it to also commit the changes on the `Gemfile.lock` file.